draft-ietf-quic-load-balancers.md

```diff
@@ -63,15 +63,14 @@ normative:
 
 --- abstract
 
-QUIC connection IDs allow continuation of connections across address/port
-4-tuple changes, and can store routing information for stateless or low-state
-load balancers.  They also can prevent linkability of connections across
-deliberate address migration through the use of protected communications between
-client and server.  This creates issues for load-balancing intermediaries.  This
-specification standardizes methods for encoding routing information given a
-small set of configuration parameters. This framework also enables offload of
-other QUIC functions to trusted intermediaries, given the explicit cooperation
-of the QUIC server.
+The QUIC protocol design is resistant to transparent packet inspection,
+injection, and modification by intermediaries. However, the server can
+explicitly cooperate with network services by agreeing to certain conventions.
```

Comment:
It's not necessarily just "agreeing to certain conventions" but also requires shared state sometime. I think that might be important to spell out, because it means there must some type of communication or shared configuration between the servers and infrastructure. I'm not quite sure how best to put that here though.